packages/Popover/src/Popover.js

```diff
@@ -441,13 +437,19 @@ class Popover extends React.Component {
       this.props.shouldKeepFocus,
       this.state.tip,
       zIndex,
-      this.ariaIdForContent
+      this.ariaIdForContent,
+      this.handleKeyDown
     );
 
     return (
       <ThemeContext.Provider value={isDark}>
         <PopoverContext.Provider value={contextValue}>
-          <sc.Popover data-pka-anchor="popover" {...moreProps} ref={this.$popover}>
+          <sc.Popover
+            data-pka-anchor="popover"
+            ref={this.$popover}
+            {...moreProps}
+            onKeyDown={callAll(this.handleKeyDown, moreProps.onKeyDown)}
```

Comment:
@nahumzs What do you think about the order of these two functions being called, when we use this order we are ensuring the Popover has the chance to catch the keyboard events first, rather than the Component that spread `onKeyDown` to the popover (eg the Listbox). I think it makes sense this way.

This is the same for line https://github.com/acl-services/paprika/pull/987/files#diff-8c0224cf91d59c94783b6e753545719c6e30a688f5ae2191f531e937f7309a56R107

Reply:
I think is ok 👍